### ISSUE TITLE
Search in Disney+ raises a `KeyError` exception

### ISSUE THREAD

**Issue by villalbamartin** [matthuisman/slyguy.addons#125]
I am getting an exception every time I use the "search" function in the Disney+ plugin.

This started happening after upgrading to OSMC running Kodi 19.3. Note that everything else works fine, meaning that I can still go through the catalogue and choose something to play manually.

The relevant exception is:

```
ERROR <general>: slyguy.disney.plus - 'data'
Traceback (most recent call last):
    File "/home/osmc/.kodi/addons/script.module.slyguy/resources/modules/slyguy/signals.py", line 41, in throwable 
        yield
    File "/home/osmc/.kodi/addons/script.module.slyguy/resources/modules/slyguy/router.py", line 121, in dispatch
        function(**params)
    File "/home/osmc/.kodi/addons/script.module.slyguy/resources/modules/slyguy/plugin.py", line 62, in decorated_function
        item = f(*args, **kwargs)
    File "/home/osmc/.kodi/addons/script.module.slyguy/resources/modules/slyguy/plugin.py", line 203, in decorated_function
        items, more_results = f(query=query, page=page, **kwargs)
    File "/home/osmc/.kodi/addons/slyguy.disney.plus/resources/lib/plugin.py", line 480, in search
        data = api.search(query)
    File "/home/osmc/.kodi/addons/slyguy.disney.plus/resources/lib/api.py", line 233, in search
        return self._json_call(endpoint)['data']['disneysearch']
    KeyError: 'data'
```

And here is the full Kodi log, in case it helps: [kodi.log](https://github.com/matthuisman/slyguy.addons/files/8007832/kodi.log)

*Edit:* From what I could see from playing with the source code, the function `self._json_call(endpoint)` is returning `{'message': 'Invalid request'}`.

**Comment by awdAvenger** [comment]
I have the same issue. I can add that my region is Norway if it matters.

I guess they changed their search APi.

**Comment by Lookens** [comment]
Same for me, i can't use the search by name but the rest is ok

**Comment by rokam** [comment]
Same here

**Comment by matthuisman** [comment]
Should be able to look into tonight or tomorrow :)

**Comment by matthuisman** [comment]
fixed with [0.11.3](https://github.com/matthuisman/slyguy.addons/commit/72cc54f3094f74d1615ac8141d4ae53a2b644bea)

should auto-update over the next hour, or can force update by following below:
https://www.matthuisman.nz/2021/07/force-update-slyguy-add-ons.html
